eventually/event/models.py

```diff
@@ -78,18 +80,39 @@ class Event(models.Model):
     latitude = models.DecimalField(max_digits=9, decimal_places=6, null=True)
     budget = models.IntegerField(null=True)
     status = models.IntegerField(default=0, choices=STATUS_CHOICES)
-    team = models.ForeignKey(Team, on_delete=models.CASCADE, null=True)
-    owner = models.ForeignKey(CustomUser, null=True)
+
+    def __repr__(self):
+        """
+        Magic method that returns string representation of
+        event instance object.
+
+        :return: event id, event team, event owner, event name, event status
+        """
+
+        return "id:{} team:{} owner:{} name:{} status:{}".format(self.id,
+                                                                 self.team.id,
+                                                                 self.owner.id,
+                                                                 self.name,
+                                                                 self.status)
 
     def __str__(self):
         """
         Magic method that returns string representation of
         event instance object.
 
-        :return: event name, event description, event status
+        :return: event id, event team, event owner, event name,
+                 event description, event duration, event status
         """
 
-        return "{} {} {}".format(self.name, self.description, self.status)
+        return "id:{} team:{} owner:{} name:{} description:{} start_at:{}\
+                duration:{} status:{}".format(self.id,
+                                              self.team.id,
```

Comment:
you have team = models.ForeignKey(CustomUser, null=True)
so add self.team.id if self.owner else None,

Reply:
team id is necessary so we do not need `if` statement in this line